templates.rst

```diff
@@ -705,6 +705,11 @@ provided by Symfony:
                     site_name: 'ACME'
                     theme: 'dark'
 
+                # add headers in response
+                headers:
+                    Content-Type: 'image/svg+xml'
+                    foo: 'bar'
+
     .. code-block:: xml
 
         <!-- config/routes.xml -->
```

Comment:
Foreach new feature, we add directive `versionadded` to tell is new. You can add it above `Checking if a Template Exists` title

```
.. versionadded:: 7.2

    The ``headers`` option was introduced in Symfony 7.2.
```

Reply:
Good information :+1: